ts/features/bonus/siciliaVola/store/actions/voucherGeneration.ts

```diff
@@ -35,9 +53,122 @@ export const svGenerateVoucherFailure = createStandardAction(
   "SV_GENERATE_VOUCHER_FAILURE"
 )<string>();
 
+/**
+ * First step of the voucher generation: category selection
+ */
+export const svGenerateVoucherSelectCategory = createStandardAction(
+  "SV_GENERATE_VOUCHER_SELECT_CATEGORY"
+)<SvBeneficiaryCategory>();
+
+/**
+ * Step for worker or sick, to confirm that the annual income is under the threshold limit
+ */
+export const svGenerateVoucherUnderThresholdIncome = createStandardAction(
+  "SV_GENERATE_VOUCHER_UNDER_THRESHOLD_INCOME"
+)<boolean>();
+
+/**
+ * Step for worker, to indicate his company data
+ */
+export const svGenerateVoucherSelectCompany = createStandardAction(
+  "SV_GENERATE_VOUCHER_SELECT_COMPANY"
+)<Company>();
+
+/**
+ * Step for sick, to indicate the hospital data
+ */
+export const svGenerateVoucherSelectHospital = createStandardAction(
+  "SV_GENERATE_VOUCHER_SELECT_HOSPITAL"
+)<Hospital>();
+
+/**
+ * Step for student, to indicate his university data
```

Comment:
Here and [above](https://github.com/pagopa/io-app/pull/3170/files#diff-ba210dc8a65d33c1f4bb3488c7d06f4341a4b23549849b4090a220d184c1c801R71) `his` can be replaced by `the` without affecting the meaning 👩 👨

Reply:
Thank you! Updated in [db10fef](https://github.com/pagopa/io-app/pull/3170/commits/db10fef4b8c031ef220832a16aa2276fa5742e03)